frontend/tests/features/wheresMyGene/landingPage.test.ts

```diff
@@ -27,7 +27,7 @@ function goToWMG(page: Page) {
   ]);
 }
 test.describe("Tests for Gene Expression page", () => {
-  test("Should verify main panel components", async ({ page }) => {
+  test.only("Should verify main panel components", async ({ page }) => {
```

Comment:
might need to remove `.only()` 😄

Reply:
Yeah, I always get caught up with this :)